LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaPhysical/PhysicalPayload.cs

```diff
@@ -59,7 +58,7 @@ public PhysicalPayload(byte[] input, bool server = false)
                 // TX_ACK That packet type is used by the gateway to send a feedback to the to inform if a downlink request has been accepted or rejected by the gateway.
                 if (Identifier == PhysicalIdentifier.TxAck)
                 {
-                    Logger.Log($"Tx ack received from gateway", LogLevel.Debug);
+                    logger?.LogDebug("Tx ack received from gateway");
```

Comment:
nit: We could avoid the safe navigation by avoid the null default parameter or/and adding an explicit check like we do already for the `input`. Constructor has only 3 usages.

Reply:
I don't think it's worth it - when I followed the path of usages I saw that it is only used from test code (or code that ended up being used in test code). It's related to the comment here: https://github.com/Azure/iotedge-lorawan-starterkit/pull/806#discussion_r750156094, apparently we have an issue tracking to move this code to the test projects.